workspaces/editors.md

```diff
@@ -230,6 +230,7 @@ RStudio:
    RUN chown -R coder:coder /var/lib/rstudio-server
    RUN echo "server-pid-file=/tmp/rstudio-server.pid" >> /etc/rstudio/rserver.conf
    RUN echo "server-data-dir=/tmp/rstudio" >> /etc/rstudio/rserver.conf
+   RUN echo "www-frame-origin=same" >> /etc/rstudio/rserver.conf
```

Comment:
@khorne3 is there a way to add this to 1.23 as well?

Reply:
Yes, we can add this to any docs. It's valid even for older versions that do not need it